spec/shared-examples/subprocess-runner.rb

```diff
@@ -32,25 +32,31 @@
     end
   end
 
-  describe "running 'ls' with a different working directory" do
+  describe "running 'ls' or 'dir' with a different working directory" do
     subject do
-      described_class.new(%w[ls]).tap do |o|
-        o.cwd = fixture_root
-      end
+      if R10K::Util::Platform.windows?
+	      described_class.new(%w[cmd /c dir]).tap do |o|
+          o.cwd = fixture_root
+        end
+	    else
+	      described_class.new(%w[ls]).tap do |o|
+          o.cwd = fixture_root
```

Comment:
some weird indentation going on here

Reply:
I saw this in my git diff, but my editor doesn't show it! Guess I'll just have to stop using emacs.

Reply:
this is still funky

Reply:
you've mixed tabs in with your spaces
`M-x untabify`